tests/trainer/test_trainer.py

```diff
@@ -1185,16 +1185,27 @@ def setup(self, stage):
     pytest.param(3, 10, 5),
 ])
 @patch("pytorch_lightning.loggers.tensorboard.TensorBoardLogger.log_metrics")
-def test_row_log_interval(log_metrics_mock, tmpdir, train_batches, max_steps, log_interval):
+def test_log_every_n_steps(log_metrics_mock, tmpdir, train_batches, max_steps, log_interval):
     model = EvalModelTemplate()
     trainer = Trainer(
         default_root_dir=tmpdir,
-        row_log_interval=log_interval,
-        log_save_interval=log_interval,
+        log_every_n_steps=log_interval,
+        flush_logs_every_n_steps=log_interval,
         limit_train_batches=train_batches,
         limit_val_batches=0,
         max_steps=max_steps,
     )
     trainer.fit(model)
     expected_calls = [call(metrics=ANY, step=s) for s in range(log_interval - 1, max_steps, log_interval)]
     log_metrics_mock.assert_has_calls(expected_calls)
+
+def test_deprecated_log_interval(tmpdir):
+    with pytest.warns(DeprecationWarning):
```

Comment:
add a match to know that we catch the proper warning... we can have multiple... :]

Reply:
also, do it one by one as this just say the at least one of the two raised warnings :]